pennylane/pauli/utils.py

```diff
@@ -49,14 +48,25 @@ def _wire_map_from_pauli_pair(pauli_word_1, pauli_word_2):
         in the Pauli word as keys, and unique integer labels as their values.
     """
     wire_labels = Wires.all_wires([pauli_word_1.wires, pauli_word_2.wires]).labels
-    wire_map = {label: i for i, label in enumerate(wire_labels)}
-    return wire_map
+    return {label: i for i, label in enumerate(wire_labels)}
 
 
 def is_pauli_word(observable):
     """
     Checks if an observable instance consists only of Pauli and Identity Operators.
 
+    A Pauli word can be either:
+
+    * A single pauli operator (see :class:`~.PauliX for an example).
+
+    * A :class:`Tensor` instance containing Pauli operators.
+
+    * A :class:`Prod` instance containing Pauli operators.
+
+    * A :class:`SProd` instance containing a Pauli operator.
```

Comment:
In the SProd case, the base could be any of the classes listed above.

```suggestion
    * A :class:`SProd` instance containing a valid Pauli word.
```